AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -490,6 +503,11 @@ - (void)setAsyncDelegate:(id<ASCollectionDelegate>)asyncDelegate
     _asyncDelegateFlags.collectionNodeCanPerformActionForItem = [_asyncDelegate respondsToSelector:@selector(collectionNode:canPerformAction:forItemAtIndexPath:sender:)];
     _asyncDelegateFlags.collectionNodePerformActionForItem = [_asyncDelegate respondsToSelector:@selector(collectionNode:performAction:forItemAtIndexPath:sender:)];
     _asyncDelegateFlags.interop = [_asyncDelegate conformsToProtocol:@protocol(ASCollectionDelegateInterop)];
+    if (_asyncDelegateFlags.interop) {
```

Comment:
Same regarding the cast -- as well as the case of only one of willDisplay vs didEndDisplaying being implemented.